docs/index.rst

```diff
@@ -94,28 +94,37 @@ Need to contact us? Please use support@vertiq.co.
 
 .. list-table:: Vertiq Module Families
    :class: borderless
+   :align: center
 
-   * - .. figure:: ./_static/module_pictures/23xx_family.png
+   * - .. figure:: ./_static/module_pictures/23xx/23xx_family.png
             :target: modules/vertiq_23xx_family.html
             :alt: Vertiq 23-XX Family
+            :height: 150
+            :width: 275
             :align: center
 
             Vertiq 23-XX Family
-     - .. figure:: ./_static/module_pictures/40xx_family.png
+     - .. figure:: ./_static/module_pictures/40xx/40xx_family.png
             :target: modules/vertiq_40xx_family.html
             :alt: Vertiq 40-XX Family
+            :height: 150
+            :width: 300
             :align: center
             
             Vertiq 40-XX Family
-     - .. figure:: ./_static/module_pictures/81xx_family.png
+     - .. figure:: ./_static/module_pictures/81xx/81xx_family.png
```

Comment:
Ok, now that the aspect ratio was stretched out it looks pretty wild. I guess if the choices are between this and the original too small one, I'd put it back to the original. The 40s and 23s look pretty stretched out for me too now, so maybe just put them back to how they were.

Reply:
should be good now. not sure why yours was stretched out, looked fine on my end. regardless, should hopefully look fine now